src/diagnostics/MOM_sum_output.F90

```diff
@@ -544,28 +542,22 @@ subroutine write_energy(u, v, h, tv, day, n, G, GV, US, CS, tracer_CSp, dt_forci
     areaTm(i,j) = G%mask2dT(i,j)*G%areaT(i,j)
   enddo ; enddo
 
-  if (GV%Boussinesq) then
-    tmp1(:,:,:) = 0.0
-    do k=1,nz ; do j=js,je ; do i=is,ie
-      tmp1(i,j,k) = h(i,j,k) * (HL2_to_kg*areaTm(i,j))
-    enddo ; enddo ; enddo
+  tmp1(:,:,:) = 0.0
+  do k=1,nz ; do j=js,je ; do i=is,ie
+    tmp1(i,j,k) = h(i,j,k) * (GV%H_to_RZ*areaTm(i,j))
+  enddo ; enddo ; enddo
+  mass_tot_RZL2 = reproducing_sum(tmp1, isr, ier, jsr, jer, sums=mass_lay, EFP_sum=mass_EFP, unscale=US%RZL2_to_kg)
 
-    mass_tot = reproducing_sum(tmp1, isr, ier, jsr, jer, sums=mass_lay, EFP_sum=mass_EFP)
-    do k=1,nz ; vol_lay(k) = (US%m_to_L**2*GV%H_to_Z/GV%H_to_kg_m2)*mass_lay(k) ; enddo
+  if (GV%Boussinesq) then
+    ! do k=1,nz ; vol_lay(k) = (GV%H_to_Z / GV%H_to_RZ) * mass_lay(k) ; enddo
```

Comment:
Keep this comment or remove?  (AFAIK R=GV%Rho0 in Boussinesq...)

Reply:
This was removed as suggested.  Thanks.